content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "CockroachDB"
+metaTitle: "CockroachDB"
+metaDescription: "Learn how to use CockroachDB with Palette Dev Engine"
+hideToC: false
+type: "appTier"
+category: ['databases']
+hiddenFromNav: false
+fullWidth: false
+logoUrl: "https://upload.wikimedia.org/wikipedia/en/thumb/3/31/Cockroach_Labs_Logo.png/220px-Cockroach_Labs_Logo.png"
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# CockroachDB
+
+CockroachDB is a [distributed SQL database](https://www.cockroachlabs.com/blog/what-is-distributed-sql/) designed for cloud-native environments, providing a reliable and scalable solution for managing data across multiple nodes and regions. Its architecture automates data replication, sharding, and rebalancing, simplifying operational tasks and enabling developers to concentrate on building their applications.
+
+With a focus on strong consistency and horizontal scalability, CockroachDB supports fast transactions and real-time data insights. Its fault-tolerant and self-healing capabilities help reduce downtime and ensure data accuracy. As a result, CockroachDB offers a stable and efficient database solution for developers looking to build robust applications in today's demanding digital landscape.
+
+
+
+# Deploy CockroachDB
+
+Palette users can deploy CockroachDB to a virtual cluster by using the following steps.
+
+## Prerequisite
+
+- A Spectro Cloud [account](https://www.spectrocloud.com/get-started/).
+
+
+- A Virtual Cluster with at least the following minimum resources.
+    - 8 CPU 
+    - 8 GB of Memory
+    - 8 GB of Storage.
+
+
+## Enablement
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+
+
+2. On the right side of the window, click on the **User Menu** and select **Switch to App Mode**.
+
+
+
+3. Navigate to the left **Main Menu** and click on **App Profiles** to create a [new App Profile](/devx/app-profile/create-app-profile/). Provide the following basic information and click **Next**.
+
+|         Parameter           | Description  |
+|-----------------------------|-----------------|
+|Application Profile Name | A custom name for the app profile.|
+|Version (optional)       | The default value is 1.0.0. You can create multiple versions of an app profile using the format **`major.minor.patch`**.
+|Description (optional)   | Description of the app profile. | 
+|Tag (optional)           |  Assign tags to the app profile.|
+ 
+
+4. Select the **CockroachDB** service and start the configuration.
+  
+
+
+5. Provide the following information to the wizard:
+  * **Name**: The application name.
```

Comment:
Another line needs to be inserted between Name and Username to make the spacing consistent in the list.